### PR TITLE
Update MddBootstrapInitialize docs

### PR DESCRIPTION
**Pull request by DrusTheAxe**

### CODE REVIEW

File: specs/dynamicdependencies/DynamicDependencies.md [hunk 1]
```diff
@@ -1263,8 +1263,12 @@ This header contains the Bootstrap API
 /// for use by the current process. If multiple packages meet the criteria the best
 /// candidate is selected.
 ///
+/// @param majorMinorVersion major and minor version of Project Reunion's framework package, encoded as `0xMMMMNNNN` where M=Major, N=Minor (e.g. 1.2 == 0x00010002).
+/// @param versionTag version tag (if any), e.g. "prerelease".
 /// @param minVersion the minimum version to use
 STDAPI MddBootstrapInitialize(
+    const UINT32 majorMinorVersion,
```
Comment (jonwis) [inline specs/dynamicdependencies/DynamicDependencies.md:1270]: Do not put const in method declarations like this.  Same goes for the `minVersion` below; take `PCPACKAGE_VERSION` maybe if it's a large structure.

File: specs/dynamicdependencies/DynamicDependencies.md [hunk 2]
```diff
@@ -1263,8 +1263,12 @@ This header contains the Bootstrap API
 /// for use by the current process. If multiple packages meet the criteria the best
 /// candidate is selected.
 ///
+/// @param majorMinorVersion major and minor version of Project Reunion's framework package, encoded as `0xMMMMNNNN` where M=Major, N=Minor (e.g. 1.2 == 0x00010002).
+/// @param versionTag version tag (if any), e.g. "prerelease".
```
Comment (jonwis) [inline specs/dynamicdependencies/DynamicDependencies.md:1267]: So this should be marked `_In_opt_`?

**Review by jonwis — CHANGES_REQUESTED** [review]

File: specs/dynamicdependencies/DynamicDependencies.md (hunk 1 above)
Comment (DrusTheAxe) [reply specs/dynamicdependencies/DynamicDependencies.md:1270]: It's not, just sizeof(INT64)

File: specs/dynamicdependencies/DynamicDependencies.md (hunk 2 above)
Comment (DrusTheAxe) [reply specs/dynamicdependencies/DynamicDependencies.md:1267]: No need. That's redundant with PCWSTR -- pointers can be null and PCWSTR data can't be changed so effectively _In_opt_

**Review by jonwis — APPROVED** [review]

**Pull request merged**